CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.52.3      2023-11-30
+
+* Reduce frequency of collector log generation
+  - Previously, real-time snapshots would generate log lines every 10 seconds,
+    which made errors hard to find
+  - Now, a single log line is printed once a minute with a summary of snapshots
+    submitted
+  - Note that `--verbose` will still log every snapshot as it's submitted
+* OpenTelemetry integration:
+  - Support `pganalyze` tracestate to set start time of the span
+  - Start time can be specified with `t` member key, the Unix time in second,
+    allowing decimals to present up to nano seconds
+  - This allows to specify a better span start and end time in case the accurate
+    time is unable to be obtained through the log, like RDS
```

Comment:
A couple of minor wording suggestions:
 - since it's the first time using it in this section, I think we should qualify RDS as "Amazon RDS" just to make it clear
 - it's precision, not accuracy, that's the issue with RDS (this feature could also address accuracy problems to some extent, but the driver is the RDS precision problems)
 - make it clearer that, by default, we're looking at log line timestamps in the logs to establish trace timings
```suggestion
  - This allows specifying a better span start and end time in case precise timestamps
    are not present in the Postgres logs, like with Amazon RDS
```
you could also add "or the log timestamps are not accurate" to point out that use case as well, but I think that's niche enough and obvious enough that it's not worth calling out here.